### ISSUE TITLE
update README to indicate that SLC is deprecated, point people to LoopBack

### ISSUE THREAD

**Issue by ashumz** [strongloop/strongloop#343]
We should update the README to indicate that Strongloop/slc is deprecated, specifically so it is also reflected on the NPM package page here: https://www.npmjs.com/package/strongloop (@rmg indicated that that page pulls from the README) 

People are still downloading Strongloop package (~6000 downloads/month) and getting error messages about things being deprecated, and can no longer use the advertised GUI (Arc) since there is no way to sign up for it. We should be deliberately pointing them to the LoopBack CLI and API Connect for the GUI component, at minimum on the README and NPM page.

**Would also be nice to have info during the installation process that indicates to the user that they are installing deprecated tech, and should instead be using LoopBack. Not sure on status of Strong-pm and the related guidance there, but @aydonat may be able to advise on the messaging around this.

**Comment by sam-github** [comment]
@ashumz can you PR an update to the README? Or at least submit the exact text you or OM want to see.

**Comment by ArKam** [comment]
It would be nice to indicate a production ready open-sourced alternative too ;-)

**Comment by sam-github** [comment]
LoopBack is open source and production ready, what are you looking for?

**Comment by ArKam** [comment]
Just looking for a process manager handling at runtime cluster grow and shrink, not looking for a full framework ;-)

**Comment by sam-github** [comment]
strong-supervisor and strong-pm continue to exist, aren't deprecated, and shouldn't be used with the `slc` wrapper anyway, not in production at least.

Check out strong-supervisor

**Comment by ArKam** [comment]
OK, I'll have a look at this process manager. Is there a reason why slc is now deprecated?
It seems to be way more powerful and feature full than any other project such as pm2 etc.

**Comment by sam-github** [comment]
slc is just a bundling together of a set of features that are available through other packages. strong-supervisor+strong-pm+apiconnect+loopback-cli together are the same as slc. We don't use the slc wrapper anymore, so aren't supporting it.

**Comment by ArKam** [comment]
ok, thanks a lot for this clear statement. I'll so use the subset packages from now.

**Comment by bestvow** [comment]
I have the same question , I can't register strongloop scl ARC , now I can use the  loopback-cli . Thanks!

**Comment by jpmyob** [comment]
Yes - please. I've watched a few videos on loopback, and they all make use of strongloop and arc, So I installed THAT, and was confused by the login / not being able to reg/login...

**Comment by rsp** [comment]
@sam-github Is the slc **still** deprecated? 
You wrote to use `strong-pm` but on https://strong-pm.io/ **and** https://github.com/strongloop/strong-pm all of the examples show `slc` command.
Since your comments here are two years old, I'd like to clarify if the entire strong-pm is wrong in the recommended commands? OR is strong-pm outdated as well? Thanks for the clarification.
